internal/cli/delete.go

```diff
@@ -14,12 +14,12 @@ var CmdDeleteApp = &cobra.Command{
 	Short: "Deletes an application",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		client, err := clients.NewEpinioClient(cmd.Flags())
+		client, err := clients.NewEpinioClient(cmd.Context(), cmd.Flags())
```

Comment:
oh cmd has a Context already. I guess this handles the cancelling from the user's side. Maybe we need to handle the case when a deployment fails?